nucypher/cli/commands/deploy.py

```diff
@@ -528,7 +530,8 @@ def allocations(general_config, actor_options, allocation_infile, gas):
 @group_actor_options
 @option_target_address
 @click.option('--value', help="Amount of tokens to transfer in the smallest denomination", type=click.INT)
-def transfer_tokens(general_config, actor_options, target_address, value):
+@click.option('--approve', help="Approve token allowance, instead of transfer", is_flag=True, default=False)
```

Comment:
1/4 Perhaps something more descriptive...? Approve by itself feels too general.

```suggestion
@click.option('--allowance-transfer', help="Transfer tokens from target address on behalf of owner", is_flag=True, default=False)
```

Reply:
^ A simpler alternative to `--allowance-transfer` (original comment) could be `--allowance` (updated comment)